notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<p>Please, split this cell. It is much easier to navigate and debug if each function is in a different cell. To help either further navigation, add cell before each function containing a header with the name of the function. For example: </p><p> </p><p><code>## plot_az</code></p><p> </p><p><code></code><code>def plot_az(....):</p><p>    ....</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Sure, I will implement this structure on the script.

Reply:
<p>done</p>